iris-mpc-gpu/Cargo.toml

```diff
@@ -9,7 +9,10 @@ repository.workspace = true
 
 [dependencies]
 bincode = "1.3.3"
-cudarc = { version = "0.12", features = ["cuda-12020", "nccl"] }
+cudarc = { git = "https://github.com/worldcoin/cudarc-fork.git", features = [
```

Comment:
Necessary to be able to pass an arbitrary amount of arguments to the kernel function

Reply:
this has been merged now, you can switch back again: https://github.com/coreylowman/cudarc/pull/321

Reply:
sadly I cannot count right and it is actually not enough for the kernel in ps/buckets and there is a follow-up PR :P
https://github.com/coreylowman/cudarc/pull/322

For this one it should be fine to use cudarc >= 0.13.3

Reply:
Done, thanks!